config/locales/de.yml

```diff
@@ -47,56 +47,56 @@ de:
         precision: 0
         format: '%u%n'
   homepage:
-    intro: "<strong>codebar</strong> is a charity that facilitates the growth of a diverse tech community by running free regular programming workshops for minority groups in tech."
+    intro: "<strong>codebar</strong> ist eine Wohltätigkeitsorganisation, die das Wachstum einer vielfältigen Tech-Community fördert, indem sie regelmäßig kostenlose Programmier-Workshops für Minderheiten in der Tech-Branche anbietet."
```

Comment:
`Wohltätigkeitsorganisation` sounds a bit odd, even if it's technically correct. I think you'd probably rather use `eine gemeinnützige Einrichtung`

Reply:
I agree that `gemeinnützig` works better than `wohltätig`. My suggestions would be `ist gemeinnützige Organisation`